docs/en/Authorization.md

```diff
@@ -236,6 +236,25 @@ When you write this code inside your permission definition provider, it finds th
 
 > Tip: It is better to check the value returned by the `GetPermissionOrNull` method since it may return null if the given permission was not defined.
 
+### Permission depending on a Condition
+
+Permission states can be managed according to a condition such as a Feature, Global Feature, or a custom condition. Any condition which implements `IPermissionStateProvider` can be added via calling `AddStateProvider()` method for **PermissionDefinition**.
+
+See an example below:
+
+```csharp
+myGroup.AddPermission("Author_Management")
+    // To depend on Global Feature
+    .AddStateProviders(new RequireGlobalFeaturesPermissionStateProvider("AuthorManagementFeature"));
```

Comment:
There are shortcut extension methods for features and global features. example:

````csharp
.RequireGlobalFeatures("AuthorManagementFeature")
````

Also, you've used `;` at end of the lines, but continued with `.` in the next lines. That code doesn't compile. You can separate examples to keep it simple